less_1_task_4.py

```diff
@@ -0,0 +1,35 @@
+'''
+Написать программу, которая генерирует в указанных пользователем границах:
+ ● случайное целое число,
+ ● случайное вещественное число,
+ ● случайный символ.
+ Для каждого из трех случаев пользователь задает свои границы диапазона.
+ Например, если надо получить случайный символ от 'a' до 'f',
+ то вводятся эти символы.
+ Программа должна вывести на экран любой символ алфавита от 'a' до 'f' включительно.
+
+
+'''
+
+from random import randint, random,  uniform
+'''
+# Генерация случайного целого в заданных границах
+a1 = int(input('введите целое число: '))
+b2 = int(input('введите второе целое число бОльшее первого: '))
+m = randint(a1,b2)
+print(f'Случайное целое число в заданных границах:{m}')
+
+
+# Генерация случайного вещественного в заданных границах
+m1 = float(input('введите дробное число: '))
+m2 = float(input('введите еще одно дробное число бОльшее первого: '))
+m3 = uniform(m1,m2)
+print(f'Случайное число с плавающей точной: {round(m3,2)}')
+'''
+# Генерация случайного символа в заданных границах
+c1 = ord(input('Введите любой символ: '))
+c2 = ord(input('Введите еще один любой символ: '))
+m2 = ((c2-c1+1)+c1)
+#print (c1,c2,m2)
+m3 = int(abs(random()* m2))
+print (f'Случайный символ: {chr(m3)}')
```

Comment:
Графический алгоритм ошибочный. Нельзя делить схему на три параллельных синхронных процесса.
Жду на разборе.

Reply:
'''
Допилил код = сейчас поиск случайного символа в заданных
границах не уходит в ноль...  соответственно
не дает ошибку. 
'''

from random import random, randint

c1 = ord(input(': '))
c2 = ord(input(': '))
print(c1, c2)
if c1<c2:
    c1,c2 = c2,c1
print (c1,c2)
m = randint(abs(c2),abs(c1))
print(chr(m))
```suggestion
print (f'Случайный символ: {chr(m3)}')
```